packages/passport/sdk/src/confirmation/confirmation.ts

```diff
@@ -180,8 +182,10 @@ export default class ConfirmationScreen {
         height: popupOptions?.height || CONFIRMATION_WINDOW_HEIGHT,
       });
       this.overlay = new Overlay(this.config.popupOverlayOptions);
-    } catch (e) {
+    } catch (error) {
       // If an error is thrown here then the popup is blocked
+      const errorMessage = error instanceof Error ? error.message : String(error);
+      trackError('passport', 'PassportLoginPopupDenied', new Error(errorMessage));
```

Comment:
A couple of things here:

1. We can omit `Passport` in the event name, as we set the module name to `passport`
2. The existing event names use camelCase, not PascalCase
3. `confirmation.ts` isn't used for login's, it's used for confirmation windows (e.g. sending a transaction, or approving a signature). You're probably after the blocked popup logic in `authManager.ts` ([here](https://github.com/immutable/ts-immutable-sdk/blob/4927bc384b51ab6905a2d57c3a20cf2b5c0d1d4e/packages/passport/sdk/src/authManager.ts#L196))

Reply:
hi @haydenfowler thanks for the review, we actually want to track `confirmation` , the naming was wrong from the beggining